Added LGBTQ colors to our apple

mmoren01 opened the pull request:
Colors fill in proportionately

Edited Test button to allow for a full apple for  demo purposes

mmoren01 committed:
Added LGBTQ colors to our apple
Colors fill in proportionately

Edited Test button to allow for a full apple for  demo purposes

emmaknor commented on components/CalorieTracker.jsx:
```diff
@@ -12,6 +12,7 @@ const CalorieTracker = () => {
   useEffect(() => {
     const {calorieGoal} = userInfo;
     const calcProgressPercent = Math.floor(calorieCount / calorieGoal * 100);
+    console.log(calcProgressPercent);
```
might want to clean this up before the next PR

emmaknor reviewed (approved):
Just one stray console log you might want to clean up in the future.

mmoren01 committed:
Merge branch 'testing' of https://github.com/Nvision-Nutrition/nvision into calorieProgressBar-LGBTColors

mmoren01 committed:
Removed console log

Changed how progress was calculated for cleaner code
Edited demo test button to reset sooner after progress chnages
